internal/testrunner/runners/system/runner.go

```diff
@@ -73,8 +73,10 @@ var (
 				logsRegexp{
 					includes: regexp.MustCompile("^Cannot index event publisher.Event"),
 					excludes: []*regexp.Regexp{
-						// this regex is excluded to ensure that logs coming from the `system` package installed by default are not taken into account
+						// This regex is excluded to ensure that logs coming from the `system` package installed by default are not taken into account
 						regexp.MustCompile(`action \[indices:data\/write\/bulk\[s\]\] is unauthorized for API key id \[.*\] of user \[.*\] on indices \[.*\], this action is granted by the index privileges \[.*\]`),
+						// Some packages fail with this error, seen when upgrading to 0.83 in https://github.com/elastic/integrations/pull/6679
+						regexp.MustCompile(`action \[indices:admin\/auto_create\] is unauthorized for API key id \[.*\] of user \[.*\] on indices \[.*\], this action is granted by the index privileges \[.*\]`),
```

Comment:
What is the reason it's OK to exclude/ignore these?

Reply:
They seem to belong to monitoring events, that are not relevant to test the integration. I am also testing this in parallel https://github.com/elastic/elastic-package/pull/1326, that is probably a better solution by now.

This is blocking us to update elastic-package in integrations.

Reply:
> They seem to belong to monitoring events, that are not relevant to test the integration.

I totally agree it's not a failure of the integration itself. IMO it would be ideal link to an issue where we are investigating the cause of these errors. Unless I am mistaken, these do look like real issues with Fleet monitoring.